walk-in-interview/src/main/webapp/new-job/script.js

```diff
@@ -192,14 +194,16 @@ function getJobDetailsFromUserInput() {
   const expiry = document.getElementById('expiry').valueAsNumber;
   const duration = document.getElementById('duration').value;
 
+  const location = await findCoordinates(postalCode);
+
   const jobDetails = {
     jobTitle: name,
     jobLocation: {
       address: address,
       postalCode: postalCode,
       region: findRegion(postalCode),
-      latitude: 1.3039, // TODO(issue/13): get these from places api
-      longitude: 103.8358,
+      latitude: location.latitude,
+      longitude: location.longitude,
```

Comment:
do we need to verify latitude and longitude? same comment for below update feature

Reply:
added check to make sure they are in Singapore